modules/cxf/src/test/java/org/mule/module/cxf/ProxySoapVersionTestCase.java

```diff
@@ -26,7 +26,7 @@
 {
     private static final HttpRequestOptions HTTP_REQUEST_OPTIONS = newOptions().method(HttpConstants.Methods.POST.name()).disableStatusCodeValidation().build();
 
-    String doGoogleSearch = "<urn:doGoogleSearch xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\" xmlns:xsd=\"http://www.w3.org/2001/XMLSchema\" xmlns:urn=\"urn:GoogleSearch\">";
+    String doGoogleSearch = "<urn:doGoogleSearch xmlns:urn=\"urn:GoogleSearch\" xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\" xmlns:xsd=\"http://www.w3.org/2001/XMLSchema\">";
```

Comment:
This is something that must be properly documented as it can break customer's apps and tests